src/explore-education-statistics-admin/src/pages/release/footnotes/__tests__/ReleaseFootnotesPage.test.tsx

```diff
@@ -0,0 +1,279 @@
+import ReleaseFootnotesPage from '@admin/pages/release/footnotes/ReleaseFootnotesPage';
+import {
+  releaseFootnotesRoute,
+  ReleaseRouteParams,
+} from '@admin/routes/releaseRoutes';
+import _footnoteService, {
+  Footnote,
+  FootnoteMeta,
+} from '@admin/services/footnoteService';
+import _permissionService from '@admin/services/permissionService';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+import { generatePath, Route, Router } from 'react-router-dom';
+
+jest.mock('@admin/services/footnoteService');
+const footnoteService = _footnoteService as jest.Mocked<
+  typeof _footnoteService
+>;
+
+jest.mock('@admin/services/permissionService');
+const permissionService = _permissionService as jest.Mocked<
+  typeof _permissionService
+>;
+
+describe('ReleaseFootnotesPage', () => {
+  const testFootnoteMeta: FootnoteMeta = {
+    subjects: {
+      'test-subject-1': {
+        subjectId: 'test-subject-1',
+        subjectName: 'Test subject 1',
+        indicators: {},
+        filters: {},
+      },
+    },
+  };
+
+  const testFootnotes: Footnote[] = [
+    { id: 'footnote-1', content: 'Footnote 1 content', subjects: {} },
+    { id: 'footnote-2', content: 'Footnote 2 content', subjects: {} },
+    { id: 'footnote-3', content: 'Footnote 3 content', subjects: {} },
+  ];
+
+  describe('with footnotes', () => {
+    beforeEach(() => {
+      permissionService.canUpdateRelease.mockResolvedValue(true);
+      footnoteService.getFootnoteMeta.mockResolvedValue(testFootnoteMeta);
+      footnoteService.getFootnotes.mockResolvedValue(testFootnotes);
+      renderPage();
+    });
+
+    test('renders the page with footnotes', async () => {
+      await waitFor(() => {
+        expect(screen.getByText('Footnotes')).toBeInTheDocument();
+      });
+
+      expect(
+        screen.getByRole('link', { name: 'Create footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        screen.getByRole('button', { name: 'Reorder footnotes' }),
+      ).toBeInTheDocument();
+
+      const footNote1 = within(
```

Comment:
Bit of a nitpick, but it should be 'footnote1' :laughing: Same for the other variables here!